docs/source/clients/cql.md

```diff
@@ -0,0 +1,104 @@
+# Using CQL
+
+## Embedded cqlsh (aka localhost)
+
+Every ScyllaDB node has an integrated `cqlsh` available. Here is an example of how it can be used.
+
+::::{tab-set}
+:::{tab-item} Any ScyllaDB Node
+```bash
+kubectl exec -it service/<sc-name>-client -c scylla -- cqlsh 
+```
+:::
+:::{tab-item} Specific ScyllaDB Node
+```bash
+kubectl exec -it pod/<sc-name>-<datacenter>-<node-index> -c scylla -- cqlsh
+```
+:::
+::::
+```text
+Connected to scylla at 127.0.0.1:9042
+[cqlsh 6.2.0 | Scylla 5.4.0-0.20231205.58a89e7a4231 | CQL spec 3.3.1 | Native protocol v4]
+Use HELP for help.
+cqlsh>
+```
+
+## Remote cqlsh
+
+This paragraph shows how to use `cqlsh` to remotely connect to a ScyllaDB node.
+It is strongly recommended to access CQL over TLS connections on port `9142` instead of unencrypted `9042`.
+Note that Scylla Operator sets up TLS certificates by default and makes them accessible in the Kubernetes API,
+so the encrypted port `9142` works by default.
+(In future releases the unencrypted port `9042` will be disabled by default, unless explicitly opted-in.)  
+
+:::{note}
+This example uses Service ClusterIP to connect to the ScyllaDB cluster. If you have configured a different networking options
+you may need to adjust this endpoint. Please refer to [discovery documentation page](./discovery.md).
+:::
+
+:::{note}
```

Comment:
nit: maybe use attention/important/tip instead?

Reply:
`attention` and `important don't render correctly for me :(
![Uploading Screenshot from 2024-03-08 14-46-25.png…]()

Reply:
Ok, looking at https://myst-parser.readthedocs.io/en/v2.0.0/syntax/admonitions.html#admonition-types I assumed they were. For the time being can you leave a comment there with an issue ref or something?

Reply:
It's a good assumption but :shrug:  (I'll leave it for now.)

@dgarcia360 do you know why `attention` and `important` admonitions don't render with our theme?

Reply:
The teme only supports the following admonitions: https://sphinx-theme.scylladb.com/stable/examples/admonitions.html

Reply:
thanks
(I kind of with it would error out instead if those are forbidden, then display them halfway)